code/modules/admin/holder2.dm

```diff
@@ -165,12 +170,28 @@ GLOBAL_PROTECT(href_token)
 		owner.holder = null
 		owner = null
 
+/datum/admins/proc/feedback_link()
+	if(!(world.time - last_forum_access_time) > 100)
```

Comment:
```suggestion
	if(!(world.time - last_forum_access_time) > 10 SECONDS)
```

Reply:
Actually this code literally does not work because `!(time - access_time)` will always return `0`, and *then* `> 100` will roll, which will always fail.

Use the cooldown defines.